orderbook/src/database/fees.rs

```diff
@@ -86,7 +86,7 @@ impl Database {
 
 #[async_trait::async_trait]
 impl Maintaining for Database {
-    async fn run_maintenance(&self) -> Result<()> {
+    async fn run_maintenance(&self, _new_block: u64) -> Result<()> {
```

Comment:
Removed it. However if the assumption is that maintenance does *not* correspond to the current block changing, then this cache should not implement Maintainable because it should always be updated when the block changes.

Reply:
I see. Maybe it would make sense rename run_maintenance to on_new_block and do this refactoring codebase wide in a separate PR?

Reply:
We don't have to make maintenance run on new blocks, maybe that isn't relevant for other components. I am not sure. Just saying that if it is, then this component shouldn't be part of it. No strong feelings either way.
If we do want to change maintenance then yes I'd do it in a separate PR.